x-pack/docs/en/rest-api/security/activate-user-profile.asciidoc

```diff
@@ -5,7 +5,10 @@
 <titleabbrev>Activate user profile</titleabbrev>
 ++++
 
-beta::[]
+NOTE: The user profile feature is designed for indirect use through {kib} and
+Elastic’s {observability}, {ents}, and {elastic-sec} solutions. Direct use is
+not supported. Elastic reserves the right to change or remove this feature in
+future releases without prior notice.
```

Comment:
@lockewritesdocs Please feel free to rephrase this admonition. I mostly copied from the [Fleet search API doc page](https://www.elastic.co/guide/en/elasticsearch/reference/current/fleet-search.html).